docs/install.md

```diff
@@ -1,6 +1,6 @@
 ## Dependencies
 
-- Draft will need a cloud-provided Kubernetes cluster to deploy your app. It's _possible_ to use Minikube, but Draft requires a wildcard domain and an ingress controller which can be difficult to set up properly on one's laptop.
+- Draft will need a cloud-provided Kubernetes cluster to deploy your app. You can also use [minikube][minikube].
```

Comment:
Can we re-word this? `kubeadm` is also another deployment tool that can be used on non-cloud-provided Kubernetes clusters and is also supported.

Something like:

> Draft will need a Kubernetes cluster to deploy your app. [Minikube](https://github.com/kubernetes/minikube), [Azure Container Services](https://azure.microsoft.com/en-us/services/container-service/) and [Google Container Engine](https://cloud.google.com/container-engine/) are a few examples that will work with Draft, but any Kubernetes cluster will do.